subsys/bluetooth/host/conn.c

```diff
@@ -339,11 +339,26 @@ void bt_conn_reset_rx_state(struct bt_conn *conn)
 	conn->rx = NULL;
 }
 
-static void bt_acl_recv(struct bt_conn *conn, struct net_buf *buf,
-			uint8_t flags)
+static void set_acl_flags(struct net_buf *buf, uint8_t value)
+{
+	if (!IS_ENABLED(CONFIG_BT_HCI_ACL_FLOW_CONTROL)) {
+		ARG_UNUSED(buf);
+		ARG_UNUSED(value);
+	}
```

Comment:
Both `buf` and `value` seem unconditionally used in this function. Is this necessary?

Reply:
whoops forgot a `return` in there. thanks